### ISSUE TITLE
Public key/PoP token data messages

### ISSUE THREAD

**Issue by celinecamacho** [dedis/student_21_pop#635]
There is no documentation specifying what messages should be sent only by the organizer or witness. For example, a close election should be sent by the organizer and a cast vote can be sent by anyone.

The following table shows all the data messages of the protocol specifications and which should be sent using the **public key** of the organizer, witness, or server. The empty slots mean that **anyone** can send the messages using their **PoP token**. 

The idea is to edit this table until we agree and then add this information to the proto-specs documentation.

| messages/data | public key 
|---|---|
| dataAddChirp | 
| dataAddChirpBroadcast | server |
| dataAddReaction | 
| dataCastVote | 
| dataCloseRollCall | organizer |
| dataCreateLao | organizer |
| dataCreateMeeting | organizer | 
| dataCreateRollCall | organizer |
| dataDeleteReaction |
| dataDeleteChirp | 
| dataDeleteChirpBroadcast | server |
| dataElect | organizer/witness
| dataElectAccept | organizer/witness
| dataPrepare | server
| dataPromise | server
| dataPropose | server
| dataAccept | server
| dataLearn | server
| dataEndElection | organizer
| dataOpenRollCall | organizer
| dataResultElection | server | 
| dataSetupElection | organizer
| dataStateLao | organizer
| dataStateMeeting | organizer
| dataUpdateLao | organizer
| dataWitnessMessage | organizer/witness

TLDR: only the add chirp, add reaction, cast vote, delete reaction and delete chirp can be sent using a PoP token. 

This is up to discussion, what do you think?

**Comment by BadrTad** [comment]
👍
